docs/api-docs/storefront/graphql/graphql-cart-metafields.mdx

```diff
@@ -0,0 +1,328 @@
+
+
+### Create a cart metafield
+
+<Callout type="info">
+The current platform limit is 250 metafields for a single cart. 
+</Callout>
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+    
+  ```graphql filename="Example mutation: Create a cart metafield" showLineNumbers copy 
+  mutation createCartMetafield {
+ 
+  cart {
+  createCartMetafield(input:{
+    cartEntityId: "4a5fd706-beb9-41b5-9ac2-01593b471168"
+    data: {
+      key: "size"
+      value: "small"
+    }
+  }) {
+    metafield {
+      id
+      entityId
+      key
+      value
+    }
+    errors {
+      ...on Error {
+        message
+      }
+    }
+  }
+```
+</Tab>
+  <Tab>
+  ```json filename="Example mutation: Create a cart metafield" showLineNumbers copy
+  {
+  "data": {
+    "cart": {
+      "createCartMetafield": {
+        "metafield": {
+          "id": "TWV0YWZpZWxkczo1Mw==",
+          "entityId": 53,
+          "key": "size",
+          "value": "small"
+        },
+        "errors": []
+      }
+    }
+  }
+}
+```
+  </Tab>
+</Tabs>
+
+### Update a cart metafield
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+  ```graphql filename="Example mutation: Update a cart metafield" showLineNumbers copy 
+  mutation updateCartMetafield {
+  cart {
+    updateCartMetafield(input:{
+      cartEntityId: "4a5fd706-beb9-41b5-9ac2-01593b471168"
+      metafieldEntityId:53
+      data: {
+        key: "Size",
+				value: "medium"
+      }
+    }) {
+      metafield {
+        id
+        entityId
+        key
+        value
+      }
+      errors {
+        ...on Error {
+          message
+        }
+      }
+    }
+  }
+}
+```
+</Tab>
+  <Tab>
+```json filename="Example mutation: Update a cart metafield" showLineNumbers copy
+ {
+  "data": {
+    "cart": {
+      "updateCartMetafield": {
+        "metafield": {
+          "id": "TWV0YWZpZWxkczo1Mw==",
+          "entityId": 53,
+          "key": "Size",
+          "value": "medium"
+        },
+        "errors": []
+      }
+    }
+  }
+}
+```
+  </Tab>
+</Tabs>
+
+### Delete a cart metafield
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+```graphql filename="Example mutation: Delete a cart metafield" showLineNumbers copy 
+ mutation deleteCartMetafield {
+ cart {
+  deleteCartMetafield(input:{
+    cartEntityId: "4a5fd706-beb9-41b5-9ac2-01593b471168"
+    metafieldEntityId: 53
+  }) {
+  errors {
+        ... on NotFoundError {
+          __typename
+          message
+        }
+        ... on ValidationError {
+          __typename
+          message
+          path
+        }
+        ... on NotAuthorisedError {
+          __typename
+        }
+      }
+    }
+  }
+}
+```
+</Tab>
+  <Tab>
+```json filename="Example mutation: Delete a cart metafield" showLineNumbers copy
+{
+  "data": {
+    "cart": {
+      "deleteCartMetafield": {
+        "errors": []
+      }
+    }
+  }
+}
+```
+  </Tab>
+</Tabs>
+
+## Resources
+
+### GraphQL documentation
+- [GraphQL Storefront API overview](/docs/storefront/graphql)
```

Comment:
add the GraphQL Storefront API auth doc here too, as well as the playground and reference pages.